### ISSUE TITLE
Pub issue 

### ISSUE THREAD

**Issue by floriangbh** [Jaguar-dart/jaguar#112]
Hello,

I'm trying to install this package on flutter project but I have the following error : 
```
Running "flutter packages get" in demo_app...
Package jaguar has no versions that match >=1.3.8 <2.0.0 derived from:
- demo_app depends on version ^1.3.8
```
I have the same issue with the 1.3.1 for exemple. 

Any idea ? 
Thank :)

**Comment by tejainece** [comment]
Hello,

Could you let me know what Dart SDK version you use? Also what other `jaguar` dependencies, you use?


Jaguar 1.3.x uses Dart 2.0.0-dev SDK.

**Comment by floriangbh** [comment]
Hello thank you for the quick response :) 

Here is a flutter output : 
```
flutter --version
Flutter 0.1.5 • channel beta • https://github.com/flutter/flutter.git
Framework • revision 3ea4d06340 (2 weeks ago) • 2018-02-22 11:12:39 -0800
Engine • revision ead227f118
Tools • Dart 2.0.0-dev.28.0.flutter-0b4f01f759
```

**Comment by floriangbh** [comment]
Fixed by the 1.3.11 ! Thank you 👏
